aggregators/lulo/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Chain } from "@defillama/sdk/build/types";
+import { FetchResult, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import customBackfill from "../../helpers/customBackfill";
+
+const endpoint = "http://api.flexlend.fi/stats"
+const startTimestamp = 1704067200; // 2024-01-01
+
+type TVLEntry = {
+  startDate: number,
+  endDate: number,
+  volume: number
+}
+
+const fetchLuloTVL = async (timestamp: number): Promise<FetchResult> => {
+
+  const options = {
+    method: "GET",
+    headers: {
+      accept: "application/json",
+      "x-lulo-api-key": String(process.env.LULO_API_KEY || ''),
+    },
+  };
+
+  const response = await fetch(`${endpoint}?timestamp=${timestamp}`, options)
+
+   const entry: TVLEntry = await response.json()
+
+    return {
+        totalVolume: entry.volume,
```

Comment:
could you add daily volume ?

Reply:
it's a lending aggregator, would that be today's total deposited (volume) minus yesterdays total deposited?

Reply:
> it's a lending aggregator, would that be today's total deposited (volume) minus yesterdays total deposited?
why it not dexs aggregators aggregators is for aggregators